packages/typescript-checker/src/group.ts

```diff
@@ -0,0 +1,64 @@
+import { Mutant, MutantTestCoverage } from '@stryker-mutator/api/core';
+
+import { SourceFiles } from './compilers/compiler';
+
+import { toPosixFileName } from './fs/tsconfig-helpers';
+
+export function createGroups(sourceFiles: SourceFiles, mutants: MutantTestCoverage[]): MutantTestCoverage[][] {
+  let mutantsWithoutGroup = [...mutants];
+  let groups: MutantTestCoverage[][] = [];
+
+  while (mutantsWithoutGroup.length) {
+    const firstMutant = mutantsWithoutGroup[0];
```

Comment:
Suggestion: use `shift` or `pop` to get a mutant from the list. That way, you also removed it (and don't need to start from `1` later).